cpp/src/Slice/Grammar.y

```diff
@@ -1674,41 +1689,33 @@ enumerator
 : ICE_IDENTIFIER
```

Comment:
Also changed the parsing rule for `enumerator`.
For some reason, it's currently returning an `EnumeratorList` with one element in it...
I updated the code to just return a normal `EnumeratPtr` instead.